packages/perseus-editor/src/styles/perseus-editor.css

```diff
@@ -1049,3 +1048,42 @@ code {
     border-bottom: 1px dashed #444;
     margin: 30px 0 10px 0;
 }
+/* Reset fieldset styles to prevent layout issues */
+fieldset {
+    margin: 0;
+    padding: 0;
+    border: none;
+    min-inline-size: 0;
+}
```

Comment:
Doing these "globally" here feels risky.  I wonder if this is something that's reset in the Frontend repo and that we need to replicate in the Storybook reset rather than as a production styling reset. 

https://github.com/Khan/perseus/blob/20d6c74d2c0c71c247e3e13a9dd957127604df34/.storybook/styles/base-page-styles.css#L1